### PR TITLE
created Stars

### PR DESCRIPTION
**Pull request by a-kovalov**
- [DEMO LINK](https://a-kovalov.github.io/layout_stars/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:last-child {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```
Comment (IvanFesenko) [inline src/style.css:42]: Move this styles to `.stars__star`
```suggestion 

```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.stars--5 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (IvanFesenko) [inline src/style.css:31]: Redudant

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Check my comments

**Comment by a-kovalov** [comment]
bug fix Stars layout - I paid close attention to comments and improved files

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+.stars--4 .stars__star {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars__star,
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:last-child {
```
Comment (mykhalenych) [inline src/style.css:33]: delete all this classes, keep only stars__star
```suggestion
.stars__star {
...
```

**Review by mykhalenych — CHANGES_REQUESTED** [review]

**Comment by a-kovalov** [comment]
the second try, I hope I did it correct
- [DEMO LINK](https://a-kovalov.github.io/layout_stars/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_stars/report/html_report/)

**Review by etojeDenys — APPROVED** [review]
LGTM

**Pull request closed without merging**